__tests__/app.test.js

```diff
@@ -56,8 +56,74 @@ describe("app", () => {
       it("404: GET - This test should respond with a 404 as the path we are searching for does not exist", () => {
         return request(app)
           .get("/api/categoriezz")
-          .expect(404);
-          });
+          .expect(404)
+          .then((response) => {
+            expect(response.statusCode).toBe(404);
+          })
+          }) 
+        
+      });
+      describe("GET /api/reviews", () => {
+        it("200: GET - Should respond with an array containing the review data.", () => {
+          return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then(({ body }) => {
+              const { reviews } = body;
+              expect(reviews).toBeInstanceOf(Array);
+              expect(reviews).toHaveLength(13);
+            });
+        });
+      });
+      describe("GET /api/reviews", () => {
```

Comment:
We should also test that the default order reviews come back in is descending by created_by. Jest-sorted can help with this